build.sh

```diff
@@ -23,6 +23,22 @@ usage()
     exit 1
 }
 
+initDistroName()
+{
+    if [ "$__BuildOS" == "Linux" ]; then
+        # Detect Distro
+        if [ "$(cat /etc/*-release | grep -cim1 ubuntu)" -eq 1 ]; then
+            export __DistroName=ubuntu
+        elif [ "$(cat /etc/*-release | grep -cim1 centos)" -eq 1 ]; then
+            export __DistroName=centos
```

Comment:
I think that is fine. We should have this function/capability be present in buildtools package (or akin). For me, I picked up the logic from https://github.com/dotnet/cli/blob/a7f438f85d5a68c8f4c61bb422c39bbf3c1e1822/scripts/common/_rid.sh. 

Needless to say, there shouldnt be multiple copies for such a function.